### DISCUSSION TITLE
Using with Windows 10

### DISCUSSION THREAD

**Discussion by ronaldborman** [GerroDen/forge-sandbox#11] [General]
Unfortunately the installation of the sandbox on Windows 10 doesn't work out of the box. Initially I had problems with Prittier and end of line characters and with the package-lock.json that is installed by default. This seems to be resolved after the latest pull requests.

For now the problems are with dot-slash notation for file paths, using bash scripts and the color output of turbo. I don't want to make a fork for a Windows specific version and am using git bash (although only recent versions work). But still wanting to use Windows to the fullest.

Are you open for the inclusion of Windows in the documentation and the occasional cross system pull request?

**Comment by GerroDen** [comment]
Hello,

Sorry that I did not see the discussion yet. This is unexpected.
We simply all use unix based systems, in particular macos in our company, so I did not check cross OS compatibility. But I would assume, that it's a small issue since most libraries are concerned about it.
The project is meant to be a start for projects.
At the moment we have most problems with react and storybook compatibility and the different handling of workspaces in build tools. That's why the build fails at the moment.
Maybe you could add issues with the error messages that you get?
The project has only a minimal common set for our company needs at the moment. We added a bunch of more to it internally.
